docs/introduction/install.rst

```diff
@@ -36,18 +36,7 @@ If you look at the :doc:`release notes <../releases/changelog>` you can see the
 Install Auspice as a developer
 ==============================
 
-This is useful for debugging, modifying the source code, or using an unpublished feature branch.
-
-.. code:: bash
-
-   # grab the GitHub auspice repo
-   git clone https://github.com/nextstrain/auspice.git
-   cd auspice
-
-   # install dependencies and make `auspice` available globally
-   npm install --global .
-
-Updating Auspice should only require pulling the new version from GitHub - it shouldn't require any ``npm`` commands. You will, however, have to re-build Auspice whenever the client-related code has changed, via ``auspice build``.
+See `DEV_DOCS.md <https://github.com/nextstrain/auspice/blob/-/DEV_DOCS.md#developer-installation>__`.
```

Comment:
Tangential question - `blob/-` redirects to `blob/master` - does `-` mean "default branch"? Is this a git thing or a GitHub thing?

Reply:
Yeah, this is a GitHub thing. Note that all of these are similar:

- https://github.com/nextstrain/auspice/blob/master/DEV_DOCS.md
- https://github.com/nextstrain/auspice/blob/-/DEV_DOCS.md
- https://github.com/nextstrain/auspice/blob/HEAD/DEV_DOCS.md
- https://github.com/nextstrain/auspice/blob/@/DEV_DOCS.md

`master` is the explicit default branch, `-` redirects to the default branch, and `HEAD`/`@` redirect to the last commit on the default branch. Also, `-` seems to only work on certain endpoints and not others, while `HEAD` is more widely supported:

- https://github.com/nextstrain/auspice/raw/-/DEV_DOCS.md
- https://github.com/nextstrain/auspice/raw/HEAD/DEV_DOCS.md

Reply:
In this case, GitHub aped Git. Git accepts `@` as shorthand for `HEAD` (which on any git remote (not just GitHub) is the default ref and in a git clone is the currently checked out ref) and `-` as shorthand for "the previous ref" (similar to `cd -` for example).

Reply:
How does "the previous ref" refer to the default _branch_ here?

Interestingly, in GitHub URLs `@` doesn't redirect to `HEAD` (but `-` does redirect to the default branch), so I wonder if there is some subtlety there on GitHub's side, or whether they really are the same.

Reply:
Well, "previous ref" doesn't make much sense in the context of GitHub, so I assume they chose to use it as a "default branch" alias instead. I'm guessing that GitHub supports `-` mainly for forward-proofing against default branch name changes, and that it was intentional and implemented closer to the frontend as a redirect (e.g. the backend never sees `-`). Whereas with `@` and `HEAD`, I'm guessing those are supported more by happenstance because their Git backend (which is not official git for most things, but their own implementation) has to support it for other reasons (e.g. git interop), and the frontend merely passes it thru to the backend.

But that's all speculation!

Reply:
(But I think reasonable speculation that fits all the observed evidence, for example, why you can use other [revision syntax](https://git-scm.com/docs/git-rev-parse#_specifying_revisions) like `@{2.years.ago}` in those URLs too.)